src/main/java/io/gatling/mojo/GatlingMojo.java

```diff
@@ -426,9 +431,10 @@ private void copyJUnitReports() throws MojoExecutionException {
         if (isIncluded && !isExcluded) {
           // check if the class is a concrete Simulation
           Class<?> clazz = testClassLoader.loadClass(className);
+          boolean isJavaSimulationClass =
+              javaSimulationClass.isPresent() && javaSimulationClass.get().isAssignableFrom(clazz);
```

Comment:
Please use `Optional#map` and `Option#orElse`.

Reply:
fixed